v2/tools/generator/internal/astmodel/package_reference.go

```diff
@@ -28,22 +28,11 @@ type PackageReference interface {
 	// IsPreview returns true if this package reference has a suffix indicating it's a preview
 	// release, false otherwise
 	IsPreview() bool
-	// TryGroupVersion returns the group and version of this reference.
-	// Returns true if the reference has a group and version, false otherwise.
-	TryGroupVersion() (string, string, bool)
-	// GroupVersion returns the group and version of this reference, triggering a panic if they aren't available
-	GroupVersion() (string, string)
 	// ImportAlias returns the import alias to use for this package reference
 	ImportAlias(style PackageImportStyle) string
```

Comment:
I think this `ImportAlias` function is also specific to the `InternalPackageReference`, so consider removing from here possibly?

Reply:
I'll do a check to see if any additional methods can be removed, but I suspect there are lingering references.